src/main/scala/MondayScala.scala

```diff
@@ -1,3 +1,36 @@
 object MondayScala extends App {
+  //Hello World!
+  println("Hello World")
+  //Assignment
+  val helloW = println("Hello World!")
+  helloW
+  //Parameters
+  def stringMethod(a: String)= {
+    println(a)
+  }
+  stringMethod("This is a test")
+  //Return Types
+  def add( num1:Int, num2:Int ) : Int = {
+    var sum:Int = 0
+    sum = num1 + num2
```

Comment:
why initialise sum to zero and then immiediately reassign it to num1 + num2, initialise it with num1 + num2 in the first place, also it should be a val